crates/biome_css_analyze/src/lint/nursery/no_unknown_unit.rs

```diff
@@ -0,0 +1,188 @@
+use biome_analyze::{context::RuleContext, declare_rule, Ast, Rule, RuleDiagnostic, RuleSource};
+use biome_console::markup;
+use biome_css_syntax::{
+    AnyCssDimension, CssFunction, CssGenericProperty, CssQueryFeaturePlain, CssSyntaxKind,
+};
+use biome_rowan::{SyntaxNodeCast, TextRange};
+
+use crate::utils::strip_vendor_prefix;
+
+const RESOLUTION_MEDIA_FEATURE_NAMES: [&str; 3] =
+    ["resolution", "min-resolution", "max-resolution"];
+
+fn is_css_hack_unit(value: &str) -> bool {
```

Comment:
Can you please document what this hack is for and why it exists?

Reply:
I have added a description of the function and the reason for using this function where it's called.